src/components/FieldMappingInput/FieldMappingInput.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { TextArea } from "@deskpro/app-sdk";
+import { FieldErrorsImpl } from "react-hook-form";
+import { DateField } from "../DateField/DateField";
+import { Field, Opportunity } from "../../api/types";
+import { DropdownSelect } from "../DropdownSelect/DropdownSelect";
+import { InputWithTitle } from "../InputWithTitle/InputWithTitle";
+import {
+  UseFormRegister,
+  UseFormSetValue,
+  UseFormWatch,
+} from "react-hook-form/dist/types";
+
+type Props = {
+  fieldsMeta: Field[];
+  errors: Partial<FieldErrorsImpl<any>>;
+  field: {
+    name: string;
+    label: string;
+    type: string;
+  };
+  people: Opportunity[];
```

Comment:
`people: Opportunity[];` is slightly confusing, perhaps just have the users query inside this component so it can fetch its own list of users? React query caches the results, so it should be fine with multiple re-renders, etc.